src/Microsoft.IdentityModel.Tokens/TokenUtilities.cs

```diff
@@ -55,5 +58,77 @@ internal static IEnumerable<SecurityKey> GetAllSigningKeys(TokenValidationParame
                 foreach (SecurityKey key in validationParameters.IssuerSigningKeys)
                     yield return key;
         }
+
+        /// <summary>
+        /// Merges claims, If an item with same type exists in both the lists, the one in claims is picked.
+        /// </summary>
+        /// <param name="claims">Collection of claims.</param>
+        /// <param name="subjectClaims">Collection of claims.</param>
+        /// <param name="replace">Tells to either replace repeating items in subjectClaims or keep them.</param>
```

Comment:
I'd change the param name to 'shouldKeepRepeatingClaims' and modify the comment slightly to "if true, repeating items in subjectClaims will be kept. Otherwise, they will be removed." (if I'm understanding this correctly)